api.go

```diff
@@ -208,7 +208,8 @@ type (
 		RecordingURL    string `json:"recordingUrl,omitempty"`
 		Mp4Url          string `json:"mp4Url,omitempty"`
 
-		Pull *StreamPull `json:"pull,omitempty"`
+		Pull      *StreamPull `json:"pull,omitempty"`
+		CreatorID CreatorID   `json:"creatorID,omitempty"`
```

Comment:
I'd keep this as a pointer since it's an optional field (omitempty is noop here)

Reply:
Thanks, updated!